src/fm_index.rs

```diff
@@ -52,7 +53,7 @@ where
     fn create(text: Vec<T>, converter: C, get_sample: impl Fn(&[u64]) -> S) -> Self {
         let text = prepare_text(text);
         let cs = sais::get_bucket_start_pos(&sais::count_chars(&text, &converter));
-        let sa = sais::sais(&text, &converter);
+        let sa = sais::build_suffix_array(&text, &converter);
```

Comment:
this is definitely a lot easier to read! It took me a while to figure out that this was a suffix array and now it's immediately obvious.